cypress/e2e/gui/detalheCliente.cy.js

```diff
@@ -0,0 +1,22 @@
+/// <reference types="cypress" />
+describe('Detalhes do cliente', () => {
+    beforeEach(() => {
+        cy.visit('/')
+    })
+    it('Campo entrada de texto deve estar desabilitado', () => {
+        cy.get('tbody tr')
+            .first()  // seleciona o primeiro elemento da lista
+            .click()  // clica no primeiro elemento da lista
+        cy.get('[data-testid="name"]')
+            .should('be.disabled')
+    })
+    it('Voltar para página inicial', () => {
+        cy.get('tbody tr')
+            .first()  // seleciona o primeiro elemento da lista
+            .click()
+        cy.get(':nth-child(7) > button')
```

Comment:
Este seletor não é legível.

Que tal algo assim?

```suggestion
        cy.contains('button', 'Back')
```

Reply:
Mais legível, não acha?

Reply:
Obrigada pela dica!